app/frontend/courses/review/CoursesReview__Root.res

```diff
@@ -746,7 +746,7 @@ let computeInitialState = () => {
 let pageTitle = (courses, courseId) => {
   let currentCourse = ArrayUtils.unsafeFind(
     course => AppRouter__Course.id(course) == courseId,
-    "Could not find currentCourse with ID " ++ courseId ++ " in CoursesReview__Root",
+    tc("current_course_not_found") ++ courseId ++ " in CoursesReview__Root",
```

Comment:
`current_course_not_found` is used in 3 files; It will be good to move it into the shared group;